lib/helpers.rb

```diff
@@ -47,6 +47,12 @@ def migrated_gz_posts
   end
 end
 
+def render_markdown md_str
+  # This is inefficient as markdown renderer should not be created every time
+  markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+  markdown.render(md_str)
+end
+
```

Comment:
I'm pushing a commit that I think helps here.

Reply:
Do you think there is a better way to render markdown in metadata? I thought about using a module or class and create only one renderer, but for that it seems I need esoteric ruby.